smoke-test/tests/read_only/test_search.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+import requests
+
+from tests.test_result_msg import add_datahub_stats
+from tests.utils import get_frontend_url, get_gms_url
+
+restli_default_headers = {
+    "X-RestLi-Protocol-Version": "2.0.0",
+}
+
+
+def _get_search_result(entity: str):
+    json = {"input": "*", "entity": entity, "start": 0, "count": 1}
+    response = requests.post(
+        f"{get_gms_url()}/entities?action=search",
+        headers=restli_default_headers,
+        json=json,
+    )
+    res_data = response.json()
+    assert res_data, f"response data was {res_data}"
+    assert res_data["value"], f"response data was {res_data}"
+    return res_data["value"]
+
+
+@pytest.mark.read_only
+@pytest.mark.parametrize(
+    "entity_type,api_name",
+    [
+        ("chart", "chart"),
+        ("dataset", "dataset"),
+        ("dashboard", "dashboard"),
+        (
+            # Task
+            "dataJob",
+            "dataJob",
+        ),
+        (
+            # Pipeline
+            "dataFlow",
+            "dataFlow",
+        ),
+        ("container", "container"),
+        ("tag", "tag"),
+        ("corpUser", "corpUser"),
```

Comment:
I'd be surprised if this is working as expected.

Here we need to use the GraphQL notion of EntityType, which is an enum defined here: https://github.com/datahub-project/datahub/blob/master/datahub-graphql-core/src/main/resources/entity.graphql